api/v1beta1/preflightvalidation_types.go

```diff
@@ -21,10 +21,12 @@ import (
 )
 
 const (
-	VerificationTrue       string = "True"
-	VerificationFalse      string = "False"
-	VerificationStageImage string = "Image"
-	VerificationStageBuild string = "Build"
+	VerificationTrue                      string = "True"
+	VerificationFalse                     string = "False"
+	VerificationStageImage                string = "Image"
+	VerificationStageBuild                string = "Build"
+	VerificationStageWaitingNextReconcile string = "WaitingNextReconcile"
+	VerificationStageFinished             string = "Finished"
```

Comment:
```suggestion
	VerificationStageRequeued string = "Requeued"
	VerificationStageDone             string = "Done"
```

Reply:
I always agree with everything you suggest :).Fixed